v2/index.html

```diff
@@ -867,99 +872,116 @@ <h4>Rhythmic Values</h4>
+                                <script type="application/json">
+                                    {
+                                        "clef": "G-2",
+                                        "data": "xxG"
+                                    }
+                                </script>
+                                <code>xx</code>
+                            </td>
+                            <td class="notation-result"></td>
+                            <td>double-sharp</td>
+                        </tr>
+                        <tr class="notation-example">
+                            <td class="notation-code">
+                                <script type="application/json">
+                                    {
+                                        "clef": "G-2",
+                                        "data": "bbB"
+                                    }
+                                </script>
+                                <code>bb</code>
+                            </td>
+                            <td class="notation-result"></td>
+                            <td>double-flat</td>
+                        </tr>
+                        <tr class="notation-example">
+                            <td class="notation-code">
+                                <script type="application/json">
+                                    {
+                                        "clef": "G-2",
+                                        "data": "nB"
+                                    }
+                                </script>
+                                <code>n</code>
+                            </td>
+                            <td class="notation-result"></td>
+                            <td>natural</td>
+                        </tr>
                     </tbody>
+
                 </table>
-            </aside>
+                <figcaption>Accidental values</figcaption>
+            </figure>
+            <p>
+                The sounding pitch of a note MAY be altered by both a key signature or an accidental. In the case of
+                a note being altered by both, the alteration of the accidental MUST be interpreted as an alteration
+                of the pitch in the key signature.
+            </p>
+            <p>
+                A note altered by an accidental SHOULD NOT be altered subsequent times within the same bar. The
+                alteration to the sounding pitch of the note is continued on subsequent notes with the same name
+                until the next <a>bar line</a>.
+            </p>
+            <p>
+                A flat accidental MUST NOT be used to lower the sounding pitch of a sharpened note by one semitone.
+                Likewise, a sharp accidental MUST NOT be used to raise the sounding pitch by one semitone. In both
+                of these cases a natural accidental MUST be used, even if this differs from the content of the
+                <a>original source</a>.
```

Comment:
I added this because it clarifies an ambiguity. If a sharpened notes in our data are lowered by both flats AND naturals, then it's impossible to know whether the note should be lowered by a semitone or a whole tone. 

So I think we need a rule that says something along these lines.

Reply:
So I think I would say that "A note with a flat after a sharpened note will be interpreted as a flat (one tone lower) and not as a natural (half a tone lower)" and stay away from concept of original source. PAE can be used outside RISM and source encoding.

Reply:
I adjusted this section to try and make it more generic. I'm not sure that it's *clearer*, though...